App/View Controllers/Posts/PostViewModel.swift

```diff
@@ -91,6 +101,9 @@ private func massageHTML(_ html: String, isIgnored: Bool) -> String {
     if isIgnored {
         document.markRevealIgnoredPostLink()
     }
+    if forumID == "267" {
+        document.addImpZoneMagicCakeCSS()
+    }
```

Comment:
What do you think about moving this to ForumTweaks? Could add a `magicCake` to `ForumTweaks.plist` and then pass in the instance of ForumTweaks here. Then we're not hardcoding forum IDs so much as keeping some metadata on them.

Reply:
Yep, that's a much better way of doing it! Done in latest commit